FirstProject/css/feedback.css

```diff
@@ -0,0 +1,162 @@
+/*Feedback HTML*/
+
+.footer_section {
+    padding: 25px 0 12px 0;
+    font-weight:bold;
+    font-size:16px;
+    color: gray;
+    text-align: right;
+}
+
+.feedback_interview_section {
+    text-align: center;
+    padding-top: 125px;
+    position: absolute;
+    width:100%;
+}
+
+.header_form_section {
+    width: 21%;
+    background-color: white;
+    height: 340px;
+}
+
+.header_background_section {
+    height: 640px;
+}
+
+h1.interview_text_area {
```

Comment:
In this situation would be ok just to style the class "interview_text_area". I used this for specificity.